app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -211,9 +211,15 @@ class MigrationsCompletedList extends React.Component {
                           {__('of')} &nbsp;<strong>{Object.keys(tasks).length} </strong>
                           {__('VMs successfully migrated.')}
                         </ListView.InfoItem>,
-                        <ListView.InfoItem key={`${plan.id}-infraMappingName`}>
-                          {plan.infraMappingName}
-                        </ListView.InfoItem>,
+                        plan.infraMappingName ? (
+                          <ListView.InfoItem key={`${plan.id}-infraMappingName`}>
+                            {plan.infraMappingName}
+                          </ListView.InfoItem>
+                        ) : (
+                          <ListView.InfoItem key={`${plan.id}-infraMappingWarning`}>
+                            <Icon type="pf" name="warning-triangle-o" /> {__('Infrastucture mapping does not exist')}
```

Comment:
Should this have a period at the end?  Same for the other one...  Its just in the complete list there is a period... but maybe that period shouldn't exist :thi\
<img width="1337" alt="screen shot 2018-07-25 at 10 20 57 am" src="https://user-images.githubusercontent.com/6640236/43206775-72dd537a-8ff4-11e8-9db4-bd51a6c87ed8.png">